content/quickstart.py

```
@@ -11,118 +11,128 @@
 
 # importing backend functions for peptide calculations
 from src.peptide_calculator import (
-    validate_peptide_sequence, 
-    apply_modification, 
-    calculate_peptide_mz, 
+    validate_peptide_sequence,
```

Ruff (0.11.9)

`src.peptide_calculator.validate_peptide_sequence` imported but unused

Remove unused import

(F401)